packages/server/src/app/handlers/handle-new-process-action.ts

```diff
@@ -0,0 +1,42 @@
+import { ethers } from 'ethers';
+import { Signature } from 'fmg-core';
+import { communication, ConcludeInstigated } from 'magmo-wallet';
+import { unreachable } from 'magmo-wallet';
+import { getProcess, startConcludeProcess } from '../../wallet/db/queries/walletProcess';
+import { updateRPSChannel } from '../services/rpsChannelManager';
+
+export async function handleNewProcessAction(ctx) {
+  const action = ctx.request.body;
+  if (!communication.isRelayableAction(action)) {
+    throw new Error(`Action ${action.type} is not valid`);
+  }
+
+  switch (action.type) {
+    case 'WALLET.COMMON.COMMITMENT_RECEIVED':
+    case 'WALLET.FUNDING.STRATEGY_PROPOSED':
+    case 'WALLET.FUNDING.STRATEGY_APPROVED':
+      return ctx;
```

Comment:
Is the path for these actions yet to be implemented? Or is this the implemented codepath?

Reply:
Yes, it's still being fleshed out.